packages/ketcher-core/src/application/render/renderStruct.ts

```diff
@@ -24,6 +27,31 @@ export class RenderStruct {
     return struct;
   }
 
+  static removeSmallAttachemntPointLabelsInModal(
+    render: Render,
+    options: any = {},
+  ) {
+    if (!options.labelInMonomerConnectionsModal) {
+      return;
+    }
+
+    render.ctab.atoms.forEach((atom: ReAtom) => {
+      if (!atom.label) {
+        return;
+      }
+      const isAttachmentPointAtom = attachmentPointRegExp.test(atom.label.text);
+
+      if (isAttachmentPointAtom) {
+        const isSmall =
+          atom.label.path.node.getBoundingClientRect().width <
+          MIN_ATTACHMENT_POINT_SIZE;
+        if (isSmall) {
+          atom.label.path.node.remove();
+        }
+      }
```

Comment:
I believe it will look better if we'll write in that manner:
```
if (!isAttachmentPointAtom) {
      return;  
}

const isSmall =
          atom.label.path.node.getBoundingClientRect().width <
          MIN_ATTACHMENT_POINT_SIZE;
          
if (isSmall) {
  atom.label.path.node.remove();
}
```